apps/storybook/src/Utilities.mdx

```diff
@@ -405,6 +405,15 @@ writing your own geometries:
 - `ScatterPointsGeometry`, used in [`ScatterPoints`](https://h5web-docs.panosc.eu/?path=/docs/visualizations-scattervis--docs) (undocumented)
 - `SurfaceMeshGeometry`, used in [`SurfaceMesh`](https://h5web-docs.panosc.eu/?path=/docs/experimental-surfacevis--docs) (experimental)
 
+### Constants
+
+#### `CAMERA_Z`
+
+The `z` position of the canvas' orthographic camera: **1000**.
+
+The camera's frustrum is configured to `{ near: 0.1, far: CAMERA_Z *2 }`, so the visible `z` range is `[-CAMERA_Z CAMERA_Z[`.
+This means that any child of `VisCanvas` positioned outside of the range `-1000 <= z < 1000` is not rendered.
+
```

Comment:
Still planning on improving/splitting up this file but thought I'd at least document the constant for now.